cross-media-measurement-api/src/main/proto/wfa/measurement/api/v2alpha/measurement.proto

```diff
@@ -147,6 +147,9 @@ message Measurement {
   // between `measurement_public_key` in `measurement_spec` and
   // `result_public_key`.
   bytes encrypted_result = 11;
+
+  // List of key value pair tags for this `Measurement`.
```

Comment:
Would you mind adding a little more detail here? How about something like:

> List of key value pair tags for this `Measurement`. These are to enable Measurement Consumers to apply labels to the `Measurement` to aid in filtering and identifying later on. Do not put sensitive data in this field.

@SanjayVas anything else we should add to the description?

Reply:
sounds great. Thank you!